unity-renderer/Assets/Scripts/MainScripts/DCL/Models/SceneAssets/AssetCatalogUrlUtils/AssetCatalogUrlsUtils.cs

```diff
@@ -1,24 +1,30 @@
 using DCL.Configuration;
 
-public static class BIWUrlUtils
+public static class AssetCatalogUrlsUtils
 {
-    public static string GetUrlSceneObjectContent() { return BIWSettings.BASE_URL_SCENE_OBJECT_CONTENT.Replace("{ENV}", GetEnvBase()); }
+    public static string GetUrlSceneObjectContent() 
+    { 
+        return AssetCatalogUrlSettings.BASE_URL_SCENE_OBJECT_CONTENT.Replace("{ENV}", GetEnvBase()); 
+    }
 
     public static string GetManifestJSON(string jsonManifest) { return "{\"manifest\":" + jsonManifest + "}"; }
     
     public static string GetBuilderAPIBaseUrl()
```

Comment:
Is this still in use? I would just delete it if not, the same goes with `BASE_URL_BUILDER_API`